src/components/Dashboard/Utilization/UtilizationItem.js

```diff
@@ -46,12 +46,27 @@ export class UtilizationItem extends React.PureComponent {
     }
 
     let actual;
+    let chartAxis;
     let chart = NOT_AVAILABLE;
     if (isLoading) {
       chart = <LoadingComponent />;
     } else if (data) {
       const chartData = data.map((val, index) => ({ x: index, y: val ? Number(val.toFixed(1)) : 0 }));
       actual = `${Math.round(data[data.length - 1])} ${unit}`;
+      if (decimalPoints > -1) {
+        chartAxis = (
+          <ChartAxis
+            dependentAxis
+            tickValues={[0, Number((maxY / 2).toFixed(decimalPoints)), Number(maxY.toFixed(decimalPoints))]}
```

Comment:
if/else block could calculate just `tickValues` which would result in one `ChartAxis` (easier to maintain if the future)